docs/0.4/examples/gameroom/walkthrough/index.md

```diff
@@ -388,3 +422,970 @@ GET /api/proposals
+  to `pending` because the proposal to create the gameroom has not yet been
+  accepted.
+
+* `created_time` is when the gameroom member entry was introduced in the table.
+
+* `updated_time` is when the gameroom member entry was last updated.
+
+At the end of the operation, the `​gameroom_member​` table looks like this:
+
+| `id` | `circuit_id` |
+| :--- | :--- |
+|`<auto generated id>`|`gameroom::acme-node-000::bubba-node-000::<UUIDv4>`|
+|`<auto generated id>`|`gameroom::acme-node-000::bubba-node-000::<UUIDv4>`|
+
+
+| `node_id` | `endpoint` | `status` |
+| :--- | :--- | :--- |
+| `acme-node-000` | `tls://splinterd-node-acme:8044` | `pending` |
+| `bubba-node-000` | `tls://splinterd-node-bubba:8044` | `pending` |
+
+
+| `created_time` | `updated_time` |
+| :--- | :--- |
+|`<time entry was created>`|`<time entry was updated>`|
+|`<time entry was created>`|`<time entry was updated>`|
+
+##### I-2.8.3. New `gameroom_service` table entry
+
+The Gameroom application authorization handler adds a new entry to the
+`​gameroom_service` table, which contains the information about the services of
+the circuit.
+
+``` sql
+CREATE TABLE IF NOT EXISTS  gameroom_service (
+  id BIGSERIAL PRIMARY KEY,
+  circuit_id TEXT NOT NULL,
+  service_id TEXT NOT NULL,
+  service_type TEXT NOT NULL,
+  allowed_nodes TEXT[][] NOT NULL,
+  arguments JSON [] NOT NULL,
+  status TEXT NOT NULL,
+  created_time TIMESTAMP NOT NULL,
+  updated_time TIMESTAMP NOT NULL,
+  FOREIGN KEY (circuit_id) REFERENCES gameroom(circuit_id) ON DELETE CASCADE
+);
+```
+
+* `circuit_id`, `service_id`, `service_type`, `arguments` and `allowed_nodes`
+  are extracted directly from the circuit proposal message received from the
+  Splinter daemon.
+
+* `status` identifies the current status of the service. In this case, it is set
+  to `pending` because the proposal to create the gameroom has not yet been
+  accepted.
+
+* `created_time` is when the gameroom service entry was introduced in the table.
+
+* `updated_time` is when the gameroom service entry was last updated.
+
+At the end of the operation, the `gameroom_service` table looks like this:
+
+| `id` | `circuit_id` |
+| :--- | :--- |
+|`<auto generated id>`|`gameroom::acme-node-000::bubba-node-000::<UUIDv4>`|
+|`<auto generated id>`|`gameroom::acme-node-000::bubba-node-000::<UUIDv4>`|
+
+
+| `service_id` | `service_type` | `allowed_nodes` |
+| :--- | :--- | :--- |
+| `gameroom_acme-node-000` | `scabbard` | `{"acme-node-000"}` |
+| `gameroom_bubba-node-000` | `scabbard` | `{"bubba-node-000"}` |
+
+
+| `arguments` | `status` |
+| :--- | :--- |
+|`"peer_services": [ "gameroom_bubba-node-000" ], "admin_keys": ....`|`pending`|
+|`"peer_services": [ "gameroom_bubba-node-000" ], "admin_keys": ....`|`pending`|
+
+
+| `created_time` | `updated_time` |
+| :--- | :--- |
+|`<time entry was created>`|`<time entry was updated>`|
+|`<time entry was created>`|`<time entry was updated>`|
+
+##### I-2.8.3. New `gameroom_service` table entry
```

Comment:
Wrong number and title: It should be I-2.3.4 (not a duplicate of the previous section).